Task1/src/main.kt

```diff
@@ -0,0 +1,4 @@
+fun main(){
+    val human: Speed = Speed(12)
```

Comment:
there is no need to specify the data type.  the compiler will detect this automatically